zonemaster/zonemaster-gui
# Small improvement of the run test page

## Purpose

Primarily to fix some contrast issues and avoid moving content.

## Context

\-

## Changes

* Change the color of the "Run" button from red to blue (fix contrast issue)
* Move undelegated test notice in advanced options (avoid form to move down when advanced options are opened, is visually closer to the related content)
* Change  link style in undelegated test notice (fix contrast issue)
* Add a decorative "Info" icon in undelegated test notice
* Slightly change the color of the advanced options toggle to match the "Run" button

![Screenshot 2023-05-11 at 09-35-43 Zonemaster](https://github.com/zonemaster/zonemaster-gui/assets/19394895/161d272d-6d68-4ffc-b3c8-a7815cbfb025)


## How to test this PR

* Looking for contrast issues on the "Run test" page (with advanced options open) using e.g. the Firefox dev tool should return nothing
* Checking that the undelegated test notice is present but has moved inside the advanced options
